tests/basic/volume_ops_test.go

```diff
@@ -2987,3 +2987,216 @@ var _ = Describe("{SharedVolFuseTest}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{FioClonedVolumeFaultInjection}", func() {
```

Comment:
Please change this TC to run in parallel on multiple volumes.

Reply:
Done and tested.